### ISSUE TITLE
splitting interleaved files with the same name

### ISSUE THREAD

**Issue by swarris** [ablab/spades#270]
When multiple interleaved libraries are used which are stored in a different location but with the same name, the file split procedure overwrites the previous split files. 
My use-case: I gave two 10x HiSeq libraries to meta-spades. The 10x longranger basic procedure by default produces a file labeled `barcoded.fastq`. This will result in a `barcoded_1.fastq` and a `barcoded_2.fastq` in metaspaces. Which will contain data from only one library.
Metaspades is able to detect it, as a `--continue` gives: 
`
== Error ==  file /home/.../split_input/barcoded_1.fastq was specified at least twice
`

**Comment by asl** [comment]
Yes, this is known issue. Note that SPAdes' built-in splitting of interleaved files is very slow as well. As a workaround I'd suggest using splitter from bbtools package.

**Comment by swarris** [comment]
Okido. In that case I'll leave this issue open. I don't mind it is somewhat slow: less steps means less chances of errors. And in my case it takes less than an hour to split up the files, followed by several days of assembly ;-)
